projects/go-style-guide/src/app/features/ui-kit/components/table-docs/components/table-actions-docs/table-actions-docs.component.html

```diff
@@ -15,6 +15,14 @@ <h2 class="go-heading-2 go-heading--no-wrap">Implementing Table Actions</h2>
       <p class="go-body-copy">
         For the table actions to appear, you must toggle them on by using <code class="code-block--inline">showTableActions</code>
       </p>
+      <p class="go-body-copy">
+        Also, the size of the button icons can be controlled via the binding <code class="code-block--inline">buttonSize</code>.
+        The design guidelines suggest using <code class="code-block--inline">buttonSize="medium"</code>, although you can choose 
+        from <code class="code-block--inline">small</code>, <code class="code-block--inline">medium</code> or <code class="code-block--inline">large</code>.
+        For more details, you can refer to the section Button Size, under Icon Button component.
+      </p>
+
+      <code [highlight]="buttonSizeExample"></code>
```

Comment:
See my other comment, but I think we can probably remove this.

Reply:
removed in latest commit.